misc/userscripts/qute-pass

```diff
@@ -134,19 +134,19 @@ def find_pass_candidates(domain, password_store_path):
     return candidates
 
 
-def _run_pass(pass_arguments, encoding):
+def _run_pass(pass_arguments, encoding, password_store):
     # The executable is conveniently named after it's mode [pass|gopass].
     pass_command = [arguments.mode]
-    process = subprocess.run(pass_command + pass_arguments, stdout=subprocess.PIPE)
+    process = subprocess.run(pass_command + pass_arguments, env={'PASSWORD_STORE_DIR': password_store}, stdout=subprocess.PIPE)
```

Comment:
Setting `env=` here will mean that any existing environment variables will be cleared - thus, things like `XDG_*_HOME`, `EDITOR`, `HOME`, exiting `PASSWORD_STORE_*` variables, ... all won't go to pass anymore.

Perhaps it'd be better to do:

```python3
env = os.environ.copy()
env['PASSWORD_STORE_DIR'] = password_store
subprocess.run(..., env=env)
```

instead?

Reply:
Good point, I agree.